examples/nodeset/server_nodeset_plcopen.c

```diff
@@ -29,18 +29,30 @@ int main(int argc, char** argv) {
     UA_Server *server = UA_Server_new();
     UA_ServerConfig_setDefault(UA_Server_getConfig(server));
 
+    UA_StatusCode retval = UA_STATUSCODE_GOOD;
+
     /* create nodes from nodeset */
-    UA_StatusCode retval = namespace_di_generated(server);
-    if(retval != UA_STATUSCODE_GOOD) {
-        UA_LOG_ERROR(UA_Log_Stdout, UA_LOGCATEGORY_SERVER, "Adding the DI namespace failed. Please check previous error output.");
-        UA_Server_delete(server);
-        return EXIT_FAILURE;
+    size_t idx = LONG_MAX;
+    UA_Server_getNamespaceByName(server, UA_STRING("http://opcfoundation.org/UA/DI/"), &idx);
+    if(idx == LONG_MAX) {
```

Comment:
Very complex. Avoid the variable and use getNamespaceByName directly in the if condition. (same to other examples).